engine/class_modules/sc_shaman.cpp

```diff
@@ -7467,11 +7475,28 @@ struct earthquake_overload_damage_t : public earthquake_damage_base_t
   earthquake_overload_damage_t( shaman_t* player, earthquake_base_t* parent ) :
     earthquake_damage_base_t( player, "earthquake_overload_damage", player->find_spell( 298765 ), parent )
   {
-    // Earthquake modifier is hardcoded rather than using effects, so we set the modifier here
-    spell_power_mod.direct = 0.3195 * player->talent.mountains_will_fall->effectN( 1 ).percent();
-    // TODO: implement spellpower coefficient extracttion from spell variable
-    // auto spell_desc = player->dbc->spell_desc_vars( this->data().id() );
-    // spell_desc.desc_vars()
+      auto coeff = 0.0;
+      auto default_option_value = (new shaman_t::options_t())->earthquake_spell_power_coefficient;
```

Comment:
Unsure if there's a nicer way to get at the default for an option than reinitializing the struct here, but this should be relatively fine I think.

Reply:
> Is there an issue with just using player->option.earthquake_spell_power_coefficient that i'm not understanding?

Mostly just trying to ensure we only utilize the option if it has been overridden. The alternative I guess would be setting the default value to 0.0 and only threading the coefficient from the option through if it is >0.